jobqueue/build.gradle

```diff
@@ -19,7 +20,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.1.3'
+        classpath 'com.android.tools.build:gradle:1.3.1'
```

Comment:
I guess I shouldn't go around trying to upgrade everything. This for some reason breaks jacoco. `./gradlew tasks` produces:
```
Could not determine the dependencies of task ':jacocoTestReport'.
...
Task with path 'testDebug' not found in root project 'jobqueue'.
```

Since this build tool bump is not really necessary I'm going to discard it.